packages/cli/src/modules/insights/database/entities/insights-by-period.ts

```diff
@@ -20,6 +28,10 @@ export class InsightsByPeriod extends BaseEntity {
 	@Column()
 	metaId: number;
 
+	@OneToOne(() => InsightsMetadata)
```

Comment:
Isn't this `ManyToOne` (many `InsightsByPeriod` map to one `InsightsMetadata`)? 🤔

Reply:
Indeed, I'll fix it